RELEASE/scripts/autoscend/auto_combat.ash

```diff
@@ -992,6 +992,15 @@ string auto_combatHandler(int round, string opp, string text)
 			return useItem($item[Jam Band Flyers]);
 		}
 	}
+	
+	if(canUse($item[chaos butterfly]) && !get_property("chaosButterflyThrown").to_boolean() && !get_property("auto_skipL12Farm").to_boolean())
+	{
+		if(canUse($item[Time-Spinner]) && auto_have_skill($skill[Ambidextrous Funkslinging]))
+		{
+			return useItems($item[chaos butterfly], $item[Time-Spinner]);
+		}
+		return useItem($item[chaos butterfly]);
+	}
```

Comment:
This needs copied into `auto_edCombatHandler` too (further down this file).
Yes they should be refactored to re-use the common code but no one has had the inclination to do so as yet (maybe when I get further down my huge list of "stuff that this codebase needs done to it" I'll consider it).

Also you should consider special combat handling for the Possibility Giant to maximize the potential of acquiring one to use in the first place (e.g. YR/sniff/pickpocket and similar (XO Skeleton hug/Ed's Lash of the Cobra etc).

Reply:
added to ed on a9a91c6

You are right on the special handling of possibility giant.

Reply:
Added to TODO list in issue #272